src/schema/fuel.schema.ts

```diff
@@ -0,0 +1,37 @@
+import z from 'zod';
+
+/* Current fuel source types covered in schema along with corresponding units for each type:
+> Home Heating and Diesel Fuel (Distillate) ~ type_api_name: dfo, units: gallon, btu
+> Natural Gas ~ type_api_name: ng, units: thousand_cubic_feet, btu
+> Propane Gas ~ type_api_name: pg, units: gallon, btu
+> Kerosene ~ type_api_name: ker, units: gallon, btu
+*/
+
+const fuelSourceUnitSchema = z.enum(['gallon', 'btu', 'thousand_cubic_feet']);
+const fuelSourceTypeSchema = z.enum(['dfo', 'ng', 'pg', 'ker']);
+
+export const unregisteredFuelRequestSchema = z.object({
+  fuel_source_type: fuelSourceTypeSchema,
+  fuel_source_unit: fuelSourceUnitSchema,
+  fuel_source_value: z.number(),
+});
+
+export type fuelSourceUnit = z.TypeOf<typeof fuelSourceUnitSchema>;
```

Comment:
Minor nitpick but exported types should start with a capital letter as well. Ex: fuelSourceUnit -> FuelSourceUnit

Reply:
Added capitalization to this and forgot to do this in the electricity schema so fixed that as well!